src/renderers/shared/fiber/__tests__/ReactIncrementalScheduling-test.js

```diff
@@ -309,37 +267,6 @@ describe('ReactIncrementalScheduling', () => {
     expect(ReactNoop.getChildren()).toEqual([span(5)]);
   });
 
-  it('does not perform animation work after time runs out', () => {
```

Comment:
Isn't this still a valuable test but for `HighPriority` instead of `AnimationPriority`?